server/spec/lib/data_import/my_anime_list/extractor/character_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe DataImport::MyAnimeList::Extractor::Character do
+  let(:guts_character) { fixture('my_anime_list/characters/guts-character.html') }
+  let(:faye_character) { fixture('my_anime_list/characters/faye-character.html') }
+  let(:kyou_character) { fixture('my_anime_list/characters/kyou-character.html') }
+  let(:sachi_character) { fixture('my_anime_list/characters/sachi-character.html') }
+  let(:shichi_character) { fixture('my_anime_list/characters/shichi-character.html') }
```

Comment:
You already specify `characters/`, you don't need to specify `sachi-character` in the filename.  Shortening the filename could fix all these issues